tag/remote/remote.go

```diff
@@ -23,13 +23,15 @@ import (
 // ConcurrentRequests defines maximum number of concurrent requests we could maintain against the registry
 var ConcurrentRequests = 32
 
+// RetryRequests is a number of retries we do in case of request failure
+var RetryRequests = 0
+
+// RetryDelay is a delay between retries of failed requests to the registry
+var RetryDelay = 5 * time.Second
```

Comment:
why not a const?

Reply:
because we need to change it on runtime @zmalik  :wink: